nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/RowMapper.java

```diff
@@ -342,20 +343,23 @@ CopyResult copy(IdWithTypes source, Serializable destParentId, String destName,
 
         public final Serializable targetId;
 
-        public final boolean isRetentionActive;
```

Comment:
i know is a low level code but shouldn't we just deprecate this property instead of removing it

Reply:
This is an internal low-level record-like class to transfer data between internal methods, so not a public API at all.